library/src/main/kotlin/org/codice/compliance/SAMLComplianceException.kt

```diff
@@ -13,41 +13,52 @@
  */
 package org.codice.compliance
 
+import org.w3c.dom.Node
+
 class SAMLComplianceException : Exception {
 
     private constructor(message: String) : super(message)
     private constructor(message: String, cause: Throwable) : super(message, cause)
 
     companion object {
-        fun create(vararg codes: SAMLSpecRefMessage, message: String, cause: Throwable? = null):
+        fun create(vararg codes: SAMLSpecRefMessage,
+                   message: String,
```

Comment:
❓ message is now required in `create`?

Reply:
It always was. The only changes here are to add the `node` and to reformat the method to put each param on a line of its own.